charts/shiny-app/values.yaml

```diff
@@ -6,15 +6,6 @@ app:
 shinyApp:
   port: 3838
   docker:
-    repository: rocker/shiny
+    repository: ""
     tag: latest
```

Comment:
You're right, it's always better to specify the tag rather than use `latest` which can be unpredictable. As discussed it's possible to override the tag, I documented the value in the README but I omitted it in the run command for simplicity sake. But I'll update it now.